rotkehlchen/chain/evm/decoding/weth/constants.py

```diff
@@ -0,0 +1,58 @@
+import typing
+from dataclasses import dataclass
+from typing import Final
+
+from rotkehlchen.assets.asset import Asset
+from rotkehlchen.constants.assets import (
+    A_ETH,
+    A_WETH,
+    A_WETH_ARB,
+    A_WETH_BASE,
+    A_WETH_OPT,
+    A_WETH_SCROLL,
+)
+from rotkehlchen.types import SUPPORTED_CHAIN_IDS, ChainID
+
+CPT_WETH: Final = 'weth'
+
+
+@dataclass
+class BaseWrappedETHAsset:
+    eth: Asset
+    weth: Asset
```

Comment:
In all the cases the eth asset is the same so you can avoid this and have only the WETH id